backend/src/main/kotlin/app/ehrenamtskarte/backend/userdata/webservice/UserImportHandler.kt

```diff
@@ -80,8 +81,8 @@ class UserImportHandler {
                 val regionId = regionsByProject[entry.get("regionKey")]
                     ?: throw UserImportException(entry.recordNumber, "Specified region not found for the current project")
 
-                val userHash = entry.get("userHash").toByteArray()
-                if (!isValidUserHash(userHash)) {
+                val userHash = entry.get("userHash")
+                if (!Argon2IdHasher.isValidUserHash(userHash)) {
                     throw UserImportException(entry.recordNumber, "Failed to validate userHash")
```

Comment:
not sure if we want to give a detailed error message, what exactly wrong with the hash ?

Reply:
i think sth like "... The hash doesn't meet the requirements." or sth. similar should be fine